.github/workflows/ci-e2e-checks.yaml

```diff
@@ -33,44 +32,63 @@ jobs:
     name: e2e-tests for K8s ${{ matrix.k8s-version }}
 
     steps:
-    - uses: actions/checkout@v2
 
     - name: Check out the repo
       uses: actions/checkout@v2
 
+    # Prepare is used to to collect information needed later in the action and expose it so it can be referenced
+    - name: Prepare
+      id: prep
+      run: |
+        echo ::set-output name=version::sha-${GITHUB_SHA::7}
```

Comment:
`Prepare` and `version` are both pretty generic here. Can we be a bit more specific so these artifacts are easily to intuit meaning from when referenced later on? Preparing what? Version of what?

Reply:
Would _Collect Release Tag_ and _releasetag_ work better?

Reply:
Yeah, I think that makes things very clear.

Reply:
In order to test that this works I will have to put back in some of the test code, push to my repo and cut a release from my repo. Next push is not going to be for your review but I expect the PR to trigger CI in tmobile repo and notify you.